<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="TMZ-DPC-2185" failures="1" tests="6" name="Runner.TestRunner" time="150.324" errors="0" timestamp="2023-11-27T16:12:56 IST" skipped="0">
  <testcase classname="Runner.TestRunner" name="runScenario" time="16.293"/>
  <testcase classname="Runner.TestRunner" name="runScenario" time="12.012"/>
  <testcase classname="Runner.TestRunner" name="runScenario" time="39.275"/>
  <testcase classname="Runner.TestRunner" name="runScenario" time="35.957"/>
  <testcase classname="Runner.TestRunner" name="runScenario" time="32.305"/>
  <testcase classname="Runner.TestRunner" name="runScenario" time="14.482">
    <failure message="expected [Recovery mail has been sent to your Email] but found []" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Recovery mail has been sent to your Email] but found []
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at Definitions.LoginDef.user_clicks_to_send_link_for_reset(LoginDef.java:109)
at âœ½.User clicks to send link for reset(file:///C:/Users/ashekharangya/Git%20Lenskart/TestDreamers_Lenskart/src/test/resources/Feature/Login.feature:31)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Runner.TestRunner -->
